tests/test_torch_agent.py

```diff
@@ -535,126 +535,123 @@ def test__add_person_tokens(self):
             "I watched C-beams glitter in the dark near the Tannhauser gate.\n"
             "All those moments will be lost in time, like tears in rain.")
         prefix = 'PRE'
-        out = agent._add_person_tokens(text, prefix, add_after_newln=False)
+        out = agent.history._add_person_tokens(text, prefix, add_after_newln=False)
         self.assertEqual(out, prefix + ' ' + text)
-        out = agent._add_person_tokens(text, prefix, add_after_newln=True)
+        out = agent.history._add_person_tokens(text, prefix, add_after_newln=True)
         idx = text.rfind('\n') + 1
         self.assertEqual(out, text[:idx] + prefix + ' ' + text[idx:])
 
-    def test_get_dialog_history(self):
+    def test_dialog_history(self):
```

Comment:
Does it make more sense to divorce this test from the agent and have it only test the History object?

Reply:
I think this test is good, but I can add more tests to make it a more comprehensive History object test